src/extension/extensionServer.ts

```diff
@@ -235,6 +237,16 @@ export class ExtensionServer implements vscode.Disposable {
                 TargetPlatformHelper.checkTargetPlatformSupport(mobilePlatformOptions.platform);
                 return mobilePlatform.beforeStartPackager()
                     .then(() => {
+                        generator.step("getReactNativeVersion");
+                        return ReactNativeProjectHelper.getReactNativePackageVersionFromNodeModules(
+                            path.resolve(request.arguments.cwd, "node_modules", "react-native")
+                            );
+                    })
+                    .catch(err => {
```

Comment:
Please make sure that this `catch` is not duplicating [that one](https://github.com/microsoft/vscode-react-native/pull/1093/files#diff-a0bb3c3c1809e7b5cf209c8263767d8fR280)

Reply:
I suppose that this error handler could be moved [here](https://github.com/RedMickey/vscode-react-native/blob/local-react-native-cli/src/extension/extensionServer.ts#L280).